plugins/outputs/http/README.md

```diff
@@ -65,6 +65,15 @@ batch format by default.
   #   # Should be set manually to "application/json" for json data_format
   #   Content-Type = "text/plain; charset=utf-8"
 
+  ## MaxIdleConns controls the maximum number of idle (keep-alive)
+  ## connections across all hosts. Zero means no limit.
+  # max_idle_conn = 0
```

Comment:
I would prefer we keep these to the defaults that were already in use rather than changing it to no limit by default. Per your feature request in #10953, I believe this should be 100?

Reply:
Sorry, I think I missed to clarify clearly, While the default used in the [DefaultTransport](https://cs.opensource.google/go/go/+/refs/tags/go1.18:src/net/http/transport.go;l=50;bpv=1;bpt=1) RoundTripper is `100`. Since the `RoundTripper` used in telegraf is defined custom, that would mean it's [no limits](https://cs.opensource.google/go/go/+/refs/tags/go1.18:src/net/http/transport.go;l=194;drc=refs%2Ftags%2Fgo1.18). So zero would mean telegraf by default stays as before